lib/providers/commands-service-provider.ts

```diff
@@ -42,11 +42,11 @@ class CommandsServiceProvider implements ICommandsServiceProvider {
 	}
 
 	public getDynamicCommands(): IFuture<string[]> {
-		return this.$screenBuilderService.allSupportedCommands(this.$screenBuilderService.generatorName);
+		return this.$screenBuilderService.allSupportedCommands(this.$injector.resolve("project").getProjectDir().wait(), this.$screenBuilderService.generatorFullName);
```

Comment:
Actually `getProjectDir`'s implementation returns `null` whenever not in a project and does not create any folders